[BUG] Build failure due to incorrect path configuration in `tsconfig.app.json`.

I've encountered a build error when attempting to run `pnpm build` with the current configuration in the repository. The error occurs due to improperly configured path aliases in `tsconfig.app.json`.

## Error Details

When executing `pnpm build`, the following error is produced:

```text
error TS5090: Non-relative paths are not allowed when 'baseUrl' is not set.
```

## Root Cause

The issue stems from the path configuration in `tsconfig.app.json`. Currently, the file contains:

```json
"paths": {
  "@/*": [
    "src/*"
  ]
}
```

This path alias configuration requires a `baseUrl` property to be properly set, which is currently missing.

--- a/tsconfig.app.json
+++ b/tsconfig.app.json
@@ -23,7 +23,7 @@
     "noUncheckedSideEffectImports": true,
     "paths": {
       "@/*": [
-        "src/*"
+        "./src/*"
       ]
     }
   },

